src/backlight/strategies/exit.py

```diff
@@ -36,6 +36,45 @@ def _exit_transaction(
     return Transaction(timestamp=exit_index, amount=-trade.sum())
 
 
+def _exit_transaction_gain_loss(
+    df: pd.DataFrame,
+    trade: pd.Series,
+    loss_threshold: float,  # Negative
```

Comment:
To align with other functions, please use positive value.
https://github.com/AlpacaDB/backlight/blob/master/src/backlight/strategies/exit.py#L234-L235